### PR TITLE
Instantiate assignment cache from factory method (FF-2113)

### PR DESCRIPTION
**Pull request by leoromanovsky**
[//]: #  (Link to the issue corresponding to this chunk of work)
Fixes: #__issue__

## Motivation and Context
[//]: #  (Why is this change required? What problem does it solve?)

Assignment cache needs to be set from the factory method.

## Description
[//]: # (Describe your changes in detail)

## How has this been tested?
[//]: # (Please describe in detail how you tested your changes)

🤔  Surprised this was not caught in tests.

[//]: # (OPTIONAL)
[//]: # (Add one or multiple labels: enhancement, refactoring, bugfix)

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -249,14 +246,16 @@ export async function init(config: IClientConfig): Promise<IEppoClient> {
...
-    // instantiate and init assignment cache if needed
+
+    // instantiate and init assignment cache
     const assignmentCache = assignmentCacheFactory({
       chromeStorage: chromeStorageIfAvailable(),
       storageKeySuffix,
     });
     if (assignmentCache instanceof HybridAssignmentCache) {
       await assignmentCache.init();
     }
```
Comment (leoromanovsky) [inline src/index.ts:257]: This assignment cache was being instantiated, initialized but not set to the singleton client.

**Review by aarsilv — APPROVED** [review]
Nice catch!

Clean up the unused import and you're good to go!

File: src/index.ts (hunk 1 above)
Comment (felipecsl) [reply src/index.ts:256]: welp my bad

**Comment by felipecsl** [comment]
@leoromanovsky I'll write a test for this as a follow up

**Pull request merged**